package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stytch",
-  "version": "1.2.3",
+  "version": "1.3.0",
```

Comment:
I'd say lets bump to 2.0.0 since we're removing an endpoint and making breaking changes to two others.